src/providers/locations-service/locations-data.mock.ts

```diff
@@ -21,7 +21,8 @@ export default [
     properties: {},
     createdAt: '2000-01-01T16:30:00.123Z',
     updatedAt: '2000-02-03T17:00:00.123Z'
-  }, {
+  },
+  {
```

Comment:
Yikes.
Is this a TSLint rule or a one you added ?
Personnaly, I prefer the previous styling (in one line), which is more consistant with the `if { } else { }` notation (and more concise).

Reply:
Yes. I prefer it that way because I find you see the separation between the objects more clearly, since the opening `{` are all on the same line (which is what the rule checks). These blocks are not concise anyway since they're huge. It's also more consistent with these kinds of arrays:

```
[
  { foo: 'bar' },
  { bar: 'foo' }
]
```

 But I don't mind turning off the rule if you prefer it the other way around.